src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_clouduseraccounts.beta.json.baseline

```diff
@@ -365,40 +298,33 @@ puts JSON.pretty_generate(response)
 #    https://console.developers.google.com/apis/api/clouduseraccounts
 # 2. This sample uses Application Default Credentials for authentication.
 #    If not already done, install the gcloud CLI from
-#    https://cloud.google.com/sdk/ and run
+#    https://cloud.google.com/sdk and run
 #    `gcloud beta auth application-default login`
 # 3. Install the Ruby client library and Application Default Credentials
 #    library by running `gem install google-api-client` and
 #    `gem install googleauth`
+
 require 'googleauth'
 require 'google/apis/clouduseraccounts_beta'
 require 'json'
 
 service = Google::Apis::ClouduseraccountsBeta::CloudUserAccountsService.new
 
-# Use Application Default Credentials for authentication when running locally.
-# For more information, see:
-# https://developers.google.com/identity/protocols/application-default-credentials
 service.authorization = \
     Google::Auth.get_application_default(['https://www.googleapis.com/auth/cloud-platform'])
 
-
-# TODO: Change placeholders below to desired parameter values for the `get_linux_authorized_keys_view` method:
-
 # Project ID for this request.
-project = '{MY-PROJECT}'
+project = ''  # TODO: Update placeholder value.
 
 # Name of the zone for this request.
-zone = '{MY-ZONE}'  # ex: 'us-central1-f'
```

Comment:
The reason is a bit nuanced, I think we discussed briefly a little more than a month ago. I'll explain in person